### PR TITLE
add Spanish translation

### PR DESCRIPTION
**Pull request by rayelward**

### CODE REVIEW

**Comment by neil-morrison44** [comment]
Awesome, thanks for this - will have a look through then get it merged ready for the next release.

File: src/i18n/locales/es/index.json [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+    },
+    "required_files": "Archivos requeridos",
+    "required_files_count": "{count} / {total} archivos",
+    "settings": "Configuración",
+    "sponsor": "Patrocinador",
+    "supports": "Soportes",
+    "supports_items": {
+      "cartridges": "Cartuchos",
+      "dock": "Base",
+      "dock_dac": "Base (Analog)",
```
Comment (neil-morrison44) [inline src/i18n/locales/es/index.json:55]: Not sure about these two - “Dock” is the product name so probably shouldn’t be translated (unless Analogue themselves translate it?)

Comment (rayelward) [reply src/i18n/locales/es/index.json:55]: Ah yes, that makes sense. Wasn't sure of the context when going through all of them. I'll update accordingly.

**Comment by neil-morrison44** [comment]
Thanks again for this - was worried for a while that no one would add any translations

**Pull request merged**